tools/i-pi/README.md

```diff
@@ -0,0 +1,17 @@
+This folder used to contain a bundled version of [i-PI](https://ipi-code.org).
+However, the bundled version was not updated and thus had become outdated.
```

Comment:
However, the bundled version was removed and i-PI is now available via PyPi ...

Reply:
Done